package.json

```diff
@@ -43,6 +43,7 @@
     "flatbush": "^3.1.0",
     "js-priority-queue": "^0.1.5",
     "pbf": "^3.1.0",
+    "string-format": "^0.5.0",
```

Comment:
If you use this, you could extend the use `Provider/URLBuilder`.
You could upgrade to 2.0.0 version.

Reply:
IMHO we don't need this dependency and can rely on a simple RegExp here (and even without a RegExp, a `replace` would be sufficient).

Reply:
Ok, I've used a string.replace instead.